src/lib/pages/frontsite/Documents/DocumentDetails/DocumentCirculation/DocumentEItems.js

```diff
@@ -18,11 +17,7 @@ export class DocumentEItems extends Component {
     return eitems.total > 0 ? (
       <>
         <Header as="h3">Access online</Header>
-        <DocumentLinks
-          eitems={eitems}
-          showMaxLinks={invenioConfig.DOCUMENTS.frontsiteMaxLinks}
-          onShowAll={this.showAll}
-        />
+        <DocumentLinks eitems={eitems} />
```

Comment:
what if we have 50 links? I see you've removed the constraint coming from the config

Reply:
Because it was unclear how I should limit links to files and other links.